src/compiler/compile/utils/contenteditable.ts

```diff
@@ -0,0 +1,60 @@
+// Utilities for managing contenteditable nodes 
+import Attribute from '../nodes/Attribute';
+import Element from '../nodes/Element';
+
+
+export const CONTENTEDITABLE_ATTR = 'contenteditable';
+
+export const CONTENTEDITABLE_BINDINGS = [
+  'textContent',
+  'innerHTML',
+  'innerText',
+];
+
+/**
+ * Returns true if node is an 'input' or 'textarea'.
+ * @param {Element} node The element to be checked
+ */
+export function isInputOrTextarea(node: Element): boolean {
```

Comment:
Can you switch to `snake_case` please for internal identifiers to match the style of the rest of the project?